Common++/src/IpAddress.cpp

```diff
@@ -201,20 +171,14 @@ namespace pcpp
 	// ~~~~~~~~~~~
 
 
-	bool IPv4Network::isValidNetmask(const std::string& netmask)
+	bool IPv4Network::isValidNetmask(const IPv4Address&	maskAddress)
```

Comment:
nit: redundant space or tab

Reply:
Thanks for catching. Btw, it's why we should really introduce clang-format 😵